README.md

```diff
@@ -137,25 +129,32 @@ Will run django tests using `--keepdb` and `--parallel`. You may pass a path to
 `make test someapp.tests.test_views`
 
 ### Adding new pypi libs
-Add the libname to either `requirements.in` or `dev-requirements.in`, then either upgrade the libs with `make upgrade` or manually compile it and then,  install.
-`pip-compile requirements.in > requirements.txt` or `make upgrade`
-`pip install -r requirements.txt`
+To add a new **backend** dependency, run `poetry add {dependency}`. If the dependency should be only available for development user append `-G dev` to the command. 
 
 ## Deployment 
 ### Setup
-This project comes with an `app.json` file, which can be used to create an app on Heroku from a GitHub repository.
+This project comes with an `render.yaml` file, which can be used to create an app on Render.com from a GitHub repository.
 
-Before deploying, please make sure you've generated an up-to-date `requirements.txt` file containing the Python dependencies. This is necessary even if you've used Docker for local runs. Do so by following [these instructions](#setup-the-backend-app).
+Before deploying, please make sure you've generated an up-to-date `poetry.lock` file containing the Python dependencies. This is necessary even if you've used Docker for local runs. Do so by following [these instructions](#setup-the-backend-app).
 
 After setting up the project, you can init a repository and push it on GitHub. If your repository is public, you can use the following button:
 
+[![Deploy to Render](https://render.com/images/deploy-to-render-button.svg)](https://render.com/deploy)
+
+
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy) 
 
 If you are in a private repository, access the following link replacing `$YOUR_REPOSITORY_LINK$` with your repository link.
 
-- `https://heroku.com/deploy?template=$YOUR_REPOSITORY_LINK$`
+- `https://render.com/deploy?repo=$YOUR_REPOSITORY_LINK$`
+
+Remember to fill the `ALLOWED_HOSTS` with the URL of your app, the default on Render.com is `appname.onrender.com`. Replace `appname` with your Render.com app name.
+
+### SendGrid
+
+To enable sending emails from your application you'll need to have a valid SendGrid account and also a valid verified sender identity. After finishing the validation process you'll be able to generate the API credentials and define the `SENDGRID_USERNAME` and `SENDGRID_PASSWORD` envvars on Render.com. 
```

Comment:
Sendgrid or Mailgun? You mention Mailgun above.

Reply:
I maintained SendGrid, but I saw no advantages in migrating to Mailgun, their pricing is pretty similar and SendGrid is backed by twillio.